packages/flutter/lib/src/widgets/image.dart

```diff
@@ -221,6 +221,14 @@ typedef ImageLoadingBuilder = Widget Function(
   ImageChunkEvent loadingProgress,
 );
 
+/// Signature for the method that is called when loading an image results in an
+/// error.
+typedef ImageErrorWidgetBuilder = Widget Function(
+  BuildContext context,
+  Object error,
```

Comment:
why Object instead of dynamic here? ImageErrorListener types this as dynamic.

Reply:
I think `ImageErrorListener` should probably be `Object`, and I'm trying to avoid introducing `dynamic` to another place in the framework. In particular for this, this gets surfaced at a higher level in the API, and it would be good to avoid people interacting with this object without casting it to something proper. In particular, if they call some method on the dynamic object without casting it, it will prevent tree shaking for all types that have that method :\

Reply:
Sounds good.